Package.swift

```diff
@@ -32,8 +32,10 @@ let package = Package(
         .target(
             name: "RediStack",
             dependencies: [
+                .product(name: "NIOCore", package: "swift-nio"),
+                .product(name: "NIOPosix", package: "swift-nio"),
+                .product(name: "NIOConcurrencyHelpers", package: "swift-nio"),
                 .product(name: "Atomics", package: "swift-atomics"),
-                .product(name: "NIO", package: "swift-nio"),
```

Comment:
Can we keep the NIO umbrella import for now. Users might depend on it... But we should not use it anywhere.